### DISCUSSION TITLE
Copy pasted recovery key wont work.

### DISCUSSION THREAD

**Discussion by EyasDmour** [ente-io/ente#2770] [Q&A]
Im on Arch Linux, using Gnome with a clipboard manager extension (been using it for a long time). trying to log into the Ente-Auth app, using my credentials which are still in the clipboard manager. and it tells me that my device isn't powerful enough to verify my password, but the app can "regenerate it" in a way that works on all devices???

just downloaded the app from the AUR the package name is ente-auth 3.1.1-1

**Comment by fo-od** [comment]
use ente-auth-bin and it should work
